client/src/features/Event/My/components/EventTabs.tsx

```diff
@@ -33,6 +33,7 @@ export const EventTabs = () => {
           events={hostEvents}
           title={UI_LABELS.ONGOING_HOST_EVENTS}
           emptyMessage={STATUS_MESSAGES.NO_HOST_EVENTS}
+          cardType="host"
```

Comment:
p3.
존재하는 상수 사용해도 좋을 것 같네용
```suggestion
          cardType={TAB_VALUES.HOST}
```

Reply:
넵 상수로 대체하여씀다

fc768c76db4b0cdbc7d1b1fdb2bc4d87b37b4e65